packages/helldoc-core/src/app/components/Search.tsx

```diff
@@ -59,7 +59,7 @@ class Search extends React.Component<{}, SearchState> {
         <input
           ref={this.inputRef}
           type="text"
-          onBlur={this.onLeaveSearchBox}
+          onBlur={e => this.onLeaveSearchBox(e)}
```

Comment:
don't need to pass `e` to function is ok .

see demo https://codesandbox.io/s/l434703q39